filebeat/input/filestream/input.go

```diff
@@ -86,7 +89,10 @@ func configure(cfg *common.Config) (loginp.Prospector, loginp.Harvester, error)
 
 	prospector, err := newFileProspector(
 		config.Paths,
-		config.IgnoreOlder,
+		common.MustNewConfigFrom(map[string]interface{}{
```

Comment:
This is somewhat hackish. @urso do you have a better idea for passing prospector configuration?

Reply:
> `newFileProspector` is private to this package. Why do you need to pass the `common.Config`?

You are right. I probably haven't realized that it is a private function. :facepalm: :facepalm: :facepalm: